lib/spack/spack/test/install.py

```diff
@@ -53,6 +53,81 @@ def test_install_and_uninstall(install_mockery, mock_fetch, monkeypatch):
         raise
 
 
+def test_pkg_attributes(install_mockery, mock_fetch, monkeypatch):
```

Comment:
If I simplify the test code some it's only 35 lines (I can drop the exception handling since that would appropriately generate a failure, the `libname` can be 2 lines, the comments are largely self evident and most can be dropped, etc.)